app/javascript/styles/modules/container.scss

```diff
@@ -1,12 +1,11 @@
 .container {
-  margin: 0 auto;
-  max-width: 1400px;
-  width: calc(100% - 4em);
-
-
   @include media(medium) { width: 94%; }
   @media print { width: 94%; }
 
+  margin: 0 auto;
+  max-width: calc(100% - 4em);
+  width: 1400px;
```

Comment:
Confirming that I'm understanding this correctly: this switches from "default is flexible to an upper bound of 1400px" to "use 1400px where possible, but otherwise set an upper bound to this flexible calculation?" (If so, clever! If not, I'm a bit confused.)

Reply:
Yep that's correct!